demo/admin/src/pages/Page.tsx

```diff
@@ -62,20 +62,11 @@ export const Page: DocumentInterface<Pick<GQLPage, "content" | "seo">, GQLPageIn
     hideInMenuIcon: FileNotMenu,
     ...createDocumentRootBlocksMethods(rootBlocks),
     ...createDocumentDependencyMethods({
-        rootBlocks,
-        prefixes: { seo: "config/" },
-        query: gql`
-            query PageDependency($id: ID!) {
-                node: page(id: $id) {
-                    id
-                    content
-                    seo
-                    pageTreeNode {
-                        id
-                        category
-                    }
-                }
-            }
-        `,
+        rootQueryName: "page",
```

Comment:
I could make some assumption about this query name using the dependency's GraphQL object type (lowercase Page -> page), but I'm not sure about this.

Reply:
However, we're not using the API Generator for page tree node documents right now